build/repo.targets

```diff
@@ -29,14 +29,44 @@
       <RepositoryRoot>$(RepositoryRoot)</RepositoryRoot>
     </ArtifactInfo>
     <FilesToExcludeFromSigning Include="$(BuildDir)Microsoft.AspNetCore.AspNetCoreModule.$(PackageVersion).nupkg" />
-  </ItemGroup>
 
+    <ArtifactInfo Include="$(AncmZipOutputPath)">
+      <ArtifactType>ZipArchive</ArtifactType>
+      <RepositoryRoot>$(RepositoryRoot)</RepositoryRoot>
+      <IsContainer>true</IsContainer>
+      <Category>shipoob</Category>
+    </ArtifactInfo>
+
+    <FilesToSign Include="x64/aspnetcore.dll" Container="$(AncmZipOutputPath)" Certificate="$(AssemblySigningCertName)" />
+    <FilesToSign Include="x86/aspnetcore.dll" Container="$(AncmZipOutputPath)" Certificate="$(AssemblySigningCertName)" />
+    <FilesToSign Include="x64/aspnetcorerh.dll" Container="$(AncmZipOutputPath)" Certificate="$(AssemblySigningCertName)" />
+    <FilesToSign Include="x86/aspnetcorerh.dll" Container="$(AncmZipOutputPath)" Certificate="$(AssemblySigningCertName)" />
+  </ItemGroup>
 
   <Target Name="PackageNativeProjects">
     <PackNuspec NuspecPath="$(MSBuildThisFileDirectory)..\nuget\AspNetCore.nuspec"
       DestinationFolder="$(BuildDir)"
       Properties="version=$(PackageVersion);Configuration=$(Configuration)"
       BasePath="$(RepositoryRoot)" />
+
+    <ItemGroup>
+      <!-- x64 -->
+      <AncmFiles Include="$(RepositoryRoot)src\AspNetCore\bin\$(Configuration)\x64\aspnetcore.dll" Link="x64\aspnetcore.dll" />
+      <AncmFiles Include="$(RepositoryRoot)src\RequestHandler\bin\$(Configuration)\x64\aspnetcorerh.dll" Link="x64\aspnetcorerh.dll" />
+      <!-- x86 -->
+      <AncmFiles Include="$(RepositoryRoot)src\AspNetCore\bin\$(Configuration)\Win32\aspnetcore.dll" Link="x86\aspnetcore.dll" />
+      <AncmFiles Include="$(RepositoryRoot)src\RequestHandler\bin\$(Configuration)\Win32\aspnetcorerh.dll" Link="x86\aspnetcorerh.dll" />
+      <!-- symbols -->
+      <AncmFiles Include="$(RepositoryRoot)src\AspNetCore\bin\$(Configuration)\x64\aspnetcore.pdb" Link="x64\aspnetcore.pdb" />
+      <AncmFiles Include="$(RepositoryRoot)src\AspNetCore\bin\$(Configuration)\Win32\aspnetcore.pdb" Link="x86\aspnetcore.pdb" />
+      <AncmFiles Include="$(RepositoryRoot)src\RequestHandler\bin\$(Configuration)\x64\aspnetcorerh.pdb" Link="x64\aspnetcorerh.pdb" />
+      <AncmFiles Include="$(RepositoryRoot)src\RequestHandler\bin\$(Configuration)\Win32\aspnetcorerh.pdb" Link="x86\aspnetcorerh.pdb" />
+    </ItemGroup>
+
+    <ZipArchive File="$(AncmZipOutputPath)"
+      Overwrite="true"
+      SourceFiles="@(AncmFiles)"
+      WorkingDirectory="$(RepositoryRoot)" />
```

Comment:
Is the working directory the output path of the zip archive? I think we should put the zip file in the artifacts/build folder

Reply:
working directory is normally the folder to zip, but since we set Link metadata on all SourceFiles, it's value is irrelevant